setup.py

```diff
@@ -17,7 +17,9 @@
     "setuptools>=41.0.0",  # to satisfy dependency constraints
     "tabulate",
     "tensorflow-probability>=0.12.0",
-    "tensorflow>=2.4.0",
+    "tensorflow>=2.4.0; platform_system!='Darwin' and platform_machine!='arm64'",
```

Comment:
Should this be `or`? I assume you want the opposite of the below? https://en.wikipedia.org/wiki/De_Morgan%27s_laws

Reply:
Oops! I will fix that later tonight.